index.js

```diff
@@ -52,6 +54,14 @@ function omit(obj, omitKey) {
     }, {});
 }
 
+function logBundleWarning(index, message, value) {
```

Comment:
instead of just warning, do we want to throw at all? i guess in the context of a live config scenario, you wouldn't want to just start breaking the server if a config is formatted incorrectly.

Reply:
This is patterned after the older [version's](https://github.com/yahoo/ycb/blob/de3975f68f98dfc80fe25e0648974ad7138eda14/index.js#L373) behavior. Seems safer to match that at least by default. If there is a use case where calling code would want to handle these errors somehow we could add an option throw.